src/app/frontend/chrome/notifications/template.html

```diff
@@ -15,11 +15,13 @@
 -->
 
 <div class="kd-notifications-container">
+  <span style="display: none"
```

Comment:
We are trying to not use styles from templates. Why do you need it? What are these 3 lines doing?

Reply:
The content is used some lines below, for the matTooltip translation. Unfortunately, there is a current limitation in the i18n angular code and it is not possible to use ICU ({VAR, select, ...}) in a i18n-matTooltip (or any i18n-*), so we need to create an invisible element for that.

It seems it is not possible either to use an ng-template element.

@ocombe do you have some advice on this problem?

Reply:
So most likely the content of i18n-matTooltip is used by the directive directly and is not analyzed as template code, which is why the ICU doesn't work?
`ng-template` should work though

you can try `ng-container` to create an invisible element

Reply:
Here is the exact description of the problem: https://github.com/angular/angular/issues/21615

And an explanation of the hack I'm using: http://chrisgriffing.com/coding/angular/2017/07/22/angular-i18n-icu-messages-attributes/

Unfortunately, that does not work with ng-container (the content is not invisible) or ng-template (the textContent is empty)